modules/javafx.controls/src/main/java/javafx/scene/control/SpinnerValueFactory.java

```diff
@@ -409,8 +413,14 @@ private T _getValue(int index) {
     /**
      * A {@link javafx.scene.control.SpinnerValueFactory} implementation designed to iterate through
      * integer values.
-     *
-     * <p>Note that the default {@link #converterProperty() converter} is implemented
+     * <p>
+     * If {@link #wrapAroundProperty wrapAround} is {@code true}, the {@code IntegerSpinnerValueFactory} will step from
```

Comment:
wrapAround still does not work in Eclipse, but this does, while providing the same visuals:

```
{@link SpinnerValueFactory#wrapAroundProperty wrapAround}
```

does it work in intellij?

Reply:
That one should work everywhere, since it is equivalent. Worth double-checking, though.

Reply:
Updated. I checked this variant with IntelliJ: While I can follow the link, it doesn't show the wrapAround field's javadoc, just the wrapAroundProperty's javadoc which is empty.

Reply:
I guess that's because IntelliJ doesn't process the javafx properties.

Reply:
that's why _some say_ eclipse is better:

![Screenshot 2024-05-07 at 08 37 11](https://github.com/openjdk/jfx/assets/107069028/2aa137f6-affc-4aa9-8bcc-c1194d505479)
![Screenshot 2024-05-07 at 08 37 19](https://github.com/openjdk/jfx/assets/107069028/277c6b7a-ab89-4634-83c5-7a6c3178f044)

Reply:
I upvoted a matching [issue in the IntelliJ bugtracker](https://youtrack.jetbrains.com/issue/IDEA-273592/JavaFX-incorrectly-navigates-to-property-field-ignores-JavaDoc). As this affects quite a few properties in javafx it would be good to see this fixed at some point.